File: apps/ios/HaruBook/Core/Models/Passage.swift
import Foundation

struct Passage: Codable, Identifiable, Hashable, Sendable {
    let id: Int
    let bookId: Int
    let orderIndex: Int
    let textEn: String
    let textKo: String?
    let audioPath: String?
    let sceneImagePath: String?
}

File: apps/android/app/src/main/java/site/smap/harubook/core/models/Passage.kt
package site.smap.harubook.core.models

import kotlinx.serialization.Serializable

@Serializable
data class Passage(
    val id: Int,
    val bookId: Int,
    val orderIndex: Int,
    val textEn: String,
    val textKo: String? = null,
    val audioPath: String? = null,
    val sceneImagePath: String? = null,
)
